### PR TITLE
{phys}[intel/2018a] SPheno v4.0.3

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.9, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/7bdf770b0536160b30a8973a9fc44681 for a full test report.

File: easybuild/easyconfigs/s/SPheno/SPheno-4.0.3-intel-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['http://www.hepforge.org/archive/spheno/']
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['43c0cb5934aa42dae0cbd7904a8895f963490817d3a1e9295c0d2884868b4da7']
+unpack_options = '--strip-components=1'
+
+skipsteps = ['configure', 'install']
+
+buildininstalldir = True
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/s/SPheno/SPheno-4.0.3-intel-2018a.eb]: just wondering, wouldn't the `MakeCp` easyblock be cleaner than `ConfigureMake` with `skipsteps = ['configure', 'install']` and `buildininstalldir = True`?

Comment (pescobar) [reply easybuild/easyconfigs/s/SPheno/SPheno-4.0.3-intel-2018a.eb]: I have also noticed that with the current approach the permissions in the installfolder are not ok

I will try the makecp easyblock

Comment (boegel) [reply easybuild/easyconfigs/s/SPheno/SPheno-4.0.3-intel-2018a.eb]: @pescobar ping?

**Comment by akesandgren** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**